.changeset/purple-hats-roll.md

```diff
@@ -0,0 +1,18 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": "tree"
```

Comment:
Would this make more sense to go in the deprecations section?

Reply:
We have been putting all the tree things, including features in tree, so I think that takes precedence.